Server/app.js

```diff
@@ -3,9 +3,11 @@ const bodyParser = require('body-parser');
 const config = require('./config/default');
 const DataLayerFactory = require('./datalayer/factory.datalayer');
 const routes = require('./config/routes');
+var cors = require('cors')
 
 const app = express();
 app.use(bodyParser.json({limit: '1mb'}));
+app.use(cors());
```

Comment:
Don't enable for all the origins. we can have a list of domains e.g. https://github.com/expressjs/cors#configuring-cors-w-dynamic-origin

Reply:
This list can be in config file

Reply:
Corrected and added in base branch. Removed from this.